src/Fabfuel/Prophiler/View/sass/screen.scss

```diff
@@ -2,6 +2,7 @@
 @import "compass/css3";
 @import "compass/utilities/general/clearfix";
 
+$c-background: #fff;
```

Comment:
In your demo, the background colors are not aligned. The button's background color is #fff, but the toolbar's is #f8f8f8